docker-compose-arm64.yml

```diff
@@ -30,7 +34,12 @@ services:
         # domain:
         # https_only: false
         # statistics_enabled: false
-        hmac_key: "CHANGE_ME!!"
+        hmac_key: "DEVELOPMENT_KEY__DO_NOT_USE"
+        invidious_companion_key: "DEVELOPMENT__KEY"
+        # Bogus companion to let Invidious start
+        invidious_companion:
```

Comment:
`invidious_companion` and `invidious_companion_key` shouldn't be required for Invidious to start